pkg/nodeconfig/nodeconfig.go

```diff
@@ -129,6 +129,43 @@ func discoverKubeAPIServerEndpoint() (string, error) {
 	return host.Status.APIServerInternalURL, nil
 }
 
+// getWICDCredentials returns the CA cert and access token associated with the WICD service account
+func getWICDCredentials() (*windows.Authentication, error) {
+	cfg, err := crclientcfg.GetConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get config to talk to kubernetes api server")
+	}
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating kubernetes clientset")
+	}
+	operatorNamespace := "openshift-windows-machine-config-operator"
```

Comment:
Can you hard code this? It is possible that the operator get deployed to another namespace. Isn't there some env var that you can get this from?

Reply:
> It is possible that the operator get deployed to another namespace

Valid point. I'm leaning towards adding the namespace to the nodeconfig struct.

I'm also getting less convinced the WICD service account credentials should be part of the nodeconfig cache, given they can change throughout the life of the cluster. I'll make this function upon the nodeconfig struct instead.

Reply:
>I'm also getting less convinced the WICD service account credentials should be part of the nodeconfig cache, given they can change throughout the life of the cluster.

While it can change through the life of a cluster,  but can it change without restarting or re-deploying the operator?

Reply:
I don't think so unless a cluster admin edits somehow

Reply:
So then why can't it be part of the nodeconfig cache?

Reply:
I moved it back as part of the nodeconfig cache after this discussion